README.md

```diff
@@ -56,35 +116,19 @@ To parse and interpret VBA macros from a document, use the vmonkey script:
 python vmonkey.py <file>
 ```
 
+To make analysis faster (see the Speedup section), do:
+
+```text
+pypy vmonkey.py -s <file>
+```
+
 If the output is too verbose and too slow, you may reduce the logging level using the
 -l option:
 
 ```text
 python vmonkey.py -l warning <file>
 ```
 
-
-Documentation:
---------------
-
-Winter is coming.
-
-
-How to Suggest Improvements, Report Issues or Contribute:
```

Comment:
Please revert this deletion, this section is needed.

Reply:
Fixed in commit c963b624223cefed44fbb08bf7e70bbca0a6bf54.